src/app/modules/group/pages/group-details/group-details.component.html

```diff
@@ -84,10 +84,18 @@
   <alg-error
     *ngIf="state.isError"
     class="dark"
-    i18n-message message="Error while loading the group info"
     [showRefreshButton]="true"
     refreshButtonType="refresh"
     (refresh)="onGroupRefreshRequired()"
-  ></alg-error>
+  >
+    <ng-container i18n>
+      <ng-container *ngIf="state.error.status === 403; else otherError;">
+        You are not allowed to view this group page.
+      </ng-container>
+      <ng-template #otherError>
+        Error while loading the group info
+      </ng-template>
+    </ng-container>
```

Comment:
i18n attribute should be set per translation, not at top level (which probably makes the top-level `ng-container` obsolete)

Reply:
Fixed